File: TaskManager/210042125_LabFinal/src/Comment.java
import java.time.LocalDateTime;

public class Comment {
    private String Id;
    String Content;
    User author;
    LocalDateTime CreatedAt;

    public Comment(String Content, User author){
        this.Content= Content;
        this.author= author;
        this.CreatedAt= LocalDateTime.now();
    }


}

File: TaskManager/LabFinal/LabFinal/Comment.cs

namespace LabFinal;
public class Comment
{
    public int Id { get; set; }
    public string Content { get; set; }
    public string Author { get; set; }
    public DateTime CreatedAt { get; set; }

    public Comment(int id, string content, string author)
    {
        Id = id;
        Content = content;
        Author = author;
        CreatedAt = DateTime.Now;
    }
}
